pkg/cassandra/version/version_test.go

```diff
@@ -91,3 +93,18 @@ func TestUnmarshalJSON(t *testing.T) {
 		)
 	}
 }
+
+func TestDeepCopy(t *testing.T) {
+	t.Run(
+		"zero value",
+		func(t *testing.T) {
+			t.Log(version.Version{}.DeepCopy())
```

Comment:
This doesn't test anything, or ever fail afaict

Reply:
It tests that a Zero value can be DeepCopied.
It originally caused a panic before I added a check for nil pointer in the implementation.
I'll leave it for now.